vnext/PropertySheets/Autolink.props

```diff
@@ -8,8 +8,9 @@
   <PropertyGroup>
     <RunAutolinkCheck Condition="'$(RunAutolinkCheck)' == ''">true</RunAutolinkCheck>
     <AutolinkCommand Condition="'$(AutolinkCommand)' == ''">npx react-native autolink-windows</AutolinkCommand>
-    <AutolinkCommandArgs Condition="'$(AutolinkCommandArgs)' == ''">--check</AutolinkCommandArgs>
     <AutolinkCommandWorkingDir Condition="'$(AutolinkCommandWorkingDir)' == ''">$([MSBuild]::GetDirectoryNameOfFileAbove($(ProjectDir), 'package.json'))</AutolinkCommandWorkingDir>
+    <AutolinkCommandArgs Condition="'$(AutolinkCommandArgs)' == '' And '$(SolutionPath)' != '' And '$(ProjectPath)' != ''">--check --sln $([MSBuild]::MakeRelative($(AutolinkCommandWorkingDir), $(SolutionPath))) --proj $([MSBuild]::MakeRelative($(AutolinkCommandWorkingDir), $(ProjectPath)))</AutolinkCommandArgs>
```

Comment:
who's setting SolutionPath and ProjectPath?

Reply:
I added this as an extra safeguard to make sure the check runs accurately for playground (and potential other multi-solution or multi-app project setups). It's conditional on SolutionPath and ProjectPath existing. Otherwise, it falls through to the default args which is to just use the result from `react-native config`.

Alternatively I could just override the args prop in playground projects, but I didn't want to make such a change in the template for all apps.

I know running msbuild directly against a project doesn't set the solution path. But  using VS or `run-windows`, it gets set.

Reply:
I'm not 100% happy with it, as we *should* be relying entirely on `react-native config`, but this was the best compromise I could come up with for multi solution setups.

Reply:
@jonthysell  thanks for the explanation. I can't find how it gets set through run-windows though (I assume VS has its own way of invoking msbuild which sets these variables up), but since we're invoking msbuild manually I would have expected us to pass /p:... or something like that